packages/server/src/followUp/send.ts

```diff
@@ -0,0 +1,118 @@
+import { FirestoreService } from '../service/firestore'
```

Comment:
Can we restructure all of this in such a way that we have the 'main' in the script directory, and other functions as members of the appropriate utility class (e.g. fn for sending follow up email to one single vote is in mg.ts?)